test/test_decomp.py

```diff
@@ -156,6 +156,8 @@ def op_assert_ref(test_case, op, test_dtype, i, orig, decomp, ref, args, kwargs)
     tol_table = {
         (torch.bfloat16, torch.ops.aten.native_layer_norm.default): 1e-5,
         (torch.float16, torch.ops.aten.native_layer_norm.default): 1e-5,
+        (torch.float16, torch.ops.aten.native_layer_norm_backward.default): 1e-3,
+        (torch.bfloat16, torch.ops.aten.native_layer_norm_backward.default): 2e-2,
```

Comment:
The tests should now be okay, it was this single op failing. I had to fix an edge case in the decomp, and add this tolerances here. This tolerances are really high, but the differences to float64 in the original implementation are already very high. It was just catching cases where, for example, orig max diff was 0.005 and decomp max diff was 0.017. I feel like the high tolerances are okay, but maybe an indication that in some cases the comparison of max diffs between orig and decomp should be on the relative, not absolute differnce.